gke-tpu-examples/training/mnist-single-tpu/src/gke-config/standard-tensorflow-bash-v4.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
```

Comment:
We shouldn't recommend creating plain pods, why not a job?

Reply:
That's the manifest to create a pod so the user can have a shell session to test the volume and verify the TPU attachment. The real training and inferencing are using jobs.